api/src/main/java/com/datastrato/gravitino/rel/Table.java

```diff
@@ -48,6 +50,11 @@ default Distribution distribution() {
     return Distributions.NONE;
   }
 
+  /** @return The indexes of the table. If no indexes are specified, Indexes.NONE is returned. */
```

Comment:
You should align your comment with your implementations, it is "Indexes.NONE" in your comment.